### ISSUE TITLE
Minified React error: Verba gui stops loading

### ISSUE THREAD

**Issue by Marc-Vreeburg** [weaviate/Verba#271]
## Description

Trying to get Verba running in Docker. Containers are build and running without errors.
When starting gui with http://ipadres_local:8000 gui hangs with browser developer tool complaining about React:

Minified React error #418
Hydration failed because the server rendered HTML didn't match the client. As a result this tree will be regenerated on the client. This can happen if a SSR-ed Client Component used:

- A server/client branch `if (typeof window !== 'undefined')`.
- Variable input such as `Date.now()` or `Math.random()` which changes each time it's called.
- Date formatting in a user's locale which doesn't match the server.
- External changing data without sending a snapshot of it along with the HTML.
- Invalid HTML tag nesting.

It can also happen if the client has a browser extension installed which messes with the HTML before React loaded.

https://react.dev/link/hydration-mismatch[missing argument]

Minified React error #423
There was an error while hydrating but React was able to recover by instead client rendering the entire root.

Failed to fetch from http://localhost:8000/api/health: TypeError: Failed to fetch

Failed to load resource: net::ERR_CONNECTION_REFUSED

Minified React error #425
Text content does not match server-rendered HTML.

## Installation

- [ ] pip install goldenverba
- [ ] pip install from source
- [x] Docker installation

If you installed via pip, please specify the version:

## Weaviate Deployment

- [ ] Local Deployment
- [x] Docker Deployment
- [ ] Cloud Deployment

## Configuration

Running Docker / Docker compose on Ubuntu

## Additional context

No problem with code before merge 2.0
I'm using a browser without extensions.

**Comment by thomashacker** [comment]
Interesting, thanks a lot for the issue! I will look into it

**Comment by thomashacker** [comment]
Hey, sorry for the late reply! 
Are you still receiving the same issue?

**Comment by thomashacker** [comment]
Interesting, thanks a lot! I just released the newest v2.1 version and made some changes to the Docker implementation. Could you retry and let me know if the error persists?

If yes, can you share more information about how you installed Verba? Thanks 🚀

**Comment by Marc-Vreeburg** [comment]
Thank you. The error has disappeared.

However now i'm having time out issues using batch vectorization with large pdf files and ingesting multiple files in one go. Looks like websocket connection times out way too early.

**Comment by thomashacker** [comment]
Yes, unfortunately, there is a known bug with the socket timeout (https://github.com/weaviate/Verba/issues/346)

Could you share the exact error message you're receiving? Thanks

**Comment by Marc-Vreeburg** [comment]
![error2](https://github.com/user-attachments/assets/f4e51eb7-2f23-43f5-a29c-9b95fbc8aa85)
![error1](https://github.com/user-attachments/assets/e6144693-492a-4133-b0fc-836547886f9b)

**Comment by thomashacker** [comment]
Thanks for the info! I think it is indeed the `timeout` error that is happening around 50 seconds while importing. Can you confirm this issue is appearing after a minute?

**Comment by Marc-Vreeburg** [comment]
No. Batch vectorization fails after exactly 300 seconds (with multiple files and/or single files) if larger than approximately 400Kb (pdf). For me to embed these files in a Docker container takes more time than 300 seconds.
